apps/exports/migrations/0016_export_config_versioning.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 3.0.5 on 2020-12-17 16:07
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('reversion', '0001_squashed_0004_auto_20160611_1202'),
+        ('exports', '0015_auto_20210114_1338'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='exportrun',
+            name='export_config',
+        ),
+        migrations.RemoveField(
+            model_name='multiprojectexportrun',
+            name='export_config',
+        ),
```

Comment:
@Rohit25negi It looks like these lines wipe the entire history since they just drop the fields that previously referenced the base file. We need to write the migrations in stages by first renaming the field and then adding the reversion field.

Sorry I should have caught this in review but I rarely pay much attention migrations and it didn't cross my mind until my local data disappeared.

I guess I'll revert this for now so we don't accidentally lose that information on any production instances like I just did on my local.

Reply:
oh!!!. Sorry I missed it too! Damn!. 
Actually I was unaware that commcare-sync is being used in any prod. Will keep that in mind going forward.

Reply:
> Actually I was unaware that commcare-sync is being used in any prod

Ah yeah, definitely good to know. I think there are 6 or 7 production instances, but thankfully we don't have any standardized deployment workflow, so it's generally unlikely that they'd get a new feature immediately after merging. Another thing that may get standardized eventually but for now I think it's very ad-hoc.